src/baseConn.ts

```diff
@@ -348,6 +390,11 @@ export class BaseRawConnection {
         break;
       }
 
+      case chars.$s: {
```

Comment:
Perhaps it's better now to call `_parseDescribeStateMessage()` in the handler of parse/execute/connection, instead of `_fallthrough()` because we won't send state descriptors arbitrarily.